ui/packages/components/src/Modal/Modal.tsx

```diff
@@ -39,39 +39,40 @@ export function Modal({
             <div
               className="fixed inset-0 z-50 bg-black/50 backdrop-blur-[2px] transition-opacity dark:bg-[#04060C]/90"
               aria-hidden="true"
-            />
-          </Dialog.Overlay>
-          {/* Full-screen container to center the panel */}
-          <div className="fixed inset-0 z-50 overflow-y-auto">
-            <motion.div
-              className={cn(
-                alignTop ? 'items-baseline' : 'items-center',
-                'flex min-h-full w-full justify-center p-6'
-              )}
-              initial={{ y: -20, opacity: 0.2 }}
-              animate={{ y: 0, opacity: 1 }}
-              exit={{
-                y: -20,
-                opacity: 0.2,
-                transition: { duration: 0.2, type: 'tween' },
-              }}
-              transition={{
-                duration: 0.15,
-                type: 'tween',
-              }}
             >
-              <Dialog.Content
-                className={cn(
-                  className,
-                  'dark:bg-slate-910 transform overflow-hidden rounded-lg bg-white shadow-xl transition-all'
-                )}
-              >
-                {(title || description) && <Header description={description}>{title}</Header>}
-                {children}
-                {footer && <Footer>{footer}</Footer>}
-              </Dialog.Content>
-            </motion.div>
-          </div>
+              {/* Full-screen container to center the panel */}
```

Comment:
I moved the contents inside the overlay to get the overflow scrolling working on the y-axis. I couldn't get it working outside, not 100% sure why but it might be something with the framer motion div.